poetry.lock

```diff
@@ -61,1012 +61,876 @@ tests_no_zope = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>
 
 [[package]]
 name = "aws-cdk.assets"
-version = "1.92.0"
+version = "1.71.0"
```

Comment:
Hard nope on this one. Maybe we should just keep this PR open to avoid dependabot recreating it every week?